packages/colour-system/scripts/generate-colors.cjs

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+const _ = require('lodash');
+const tinycolor = require('tinycolor2');
+const fs = require('fs-extra');
+const path = require('path');
+const { render } = require('ejs');
+
+const figmaConfig = {
+  baseUrl: 'https://api.figma.com',
+  fileKey: '6lQm8QMegMx1CHchbzkHyc',
+  headers: new Headers({ 'X-Figma-Token': process.env.FIGMA_ACCESS_TOKEN }),
+};
+
+/**
+ * Gets the UW Colour System figma styles.
+ */
+async function getStyles() {
+  const resp = await fetch(`${figmaConfig.baseUrl}/v1/files/${figmaConfig.fileKey}/styles`, {
+    headers: figmaConfig.headers,
+  });
+  const data = await resp.json();
+  if (data.status === 403 && data.err === 'Invalid token') {
+    throw new Error(
+      'An invalid token was used. Follow the Auth Guide (https://git.io/Je87i), and try again.'
+    );
+  }
+
+  return data.meta.styles
+    .filter(s => !s.name.toLowerCase().includes('white')) // temporary
+    .filter(s => !s.name.toLowerCase().includes('midnight')) // temporary
```

Comment:
Very minor, but this would be better in a single `filter` statement instead of filtering twice.